BUILDING.md

```diff
@@ -119,3 +119,50 @@ See the [setup guide for Kubernetes](QUICKSTART-EKS.md) or the [setup guide for
 ## Publish your image
 
 See the [PUBLISHING](PUBLISHING.md) guide for information on deploying Bottlerocket images and repositories.
+
+## Building out-of-tree kernel modules
+
+To further extend Bottlerocket, you may want to build extra kernel modules.
+This section will be filled out more in the future, but the first step is to download the "kmod kit" that contains the kernel headers and toolchain you'll need to use.
+
+### Downloading the Bottlerocket kmod kit
```

Comment:
nit: it might read better without "Bottlerocket" in these titles, it feels a bit redundant